File: src/main/java/com/daveme/chocolateCakePHP/NamespaceColumn.java
package com.daveme.chocolateCakePHP;

import org.jetbrains.annotations.Nullable;

import javax.swing.table.TableModel;

public class NamespaceColumn extends com.intellij.util.ui.ColumnInfo<PluginEntry, String> {

    public NamespaceColumn(String name) {
        super(name);
    }

    @Nullable
    @Override
    public String valueOf(PluginEntry pluginEntry) {
        return pluginEntry.getNamespace();
    }
}

File: src/main/kotlin/com/daveme/chocolateCakePHP/ui/NamespaceColumn.kt
package com.daveme.chocolateCakePHP.ui

import com.daveme.chocolateCakePHP.cake.PluginEntry
import com.intellij.util.ui.ColumnInfo

class NamespaceColumn(name: String) :
    ColumnInfo<PluginEntry, String>(name) {

    override fun valueOf(pluginEntry: PluginEntry): String {
        return pluginEntry.namespace
    }

}